### PR TITLE
Update kubectl generator for 1.24

### PR DESCRIPTION
**Pull request by liggitt**
xref https://github.com/kubernetes/website/pull/33676 and https://github.com/kubernetes/website/issues/33674

### CODE REVIEW

File: gen-kubectldocs/generators/gen.go [hunk 1]
```diff
@@ -252,6 +252,7 @@ func WriteCommandFile(manifest *Manifest, t *template.Template, params TopLevelC
 		">", "&gt;",
 		"[", "<span>[</span>",
 		"]", "<span>]</span>",
+		"\n", "<br>",
```
Comment (liggitt) [inline gen-kubectldocs/generators/gen.go:255]: generator choked on usage strings that included newlines

**Comment by tengqm** [comment]
/approve
Thanks.

**Comment by sftim** [comment]
/approve

I'm not much cop with Golang so won't LGTM.

**Pull request merged**